src/fiatExchanges/FiatExchangeAmount.tsx

```diff
@@ -166,6 +169,8 @@ function FiatExchangeAmount({ route }: Props) {
   }
 
   function onPressContinue() {
+    const maxWithdrawAmount = balances[currency]?.minus(estimatedFee) || new BigNumber(0)
```

Comment:
might need to subtract a buffer in case the estimate is low, maybe 1.5x the estimated fee?

Reply:
also on the off chance this is negative it might be worth taking a max here

Reply:
This logic guards against the exact same scenario (not enough gas fees to send the $$) in the Send flow. I'd prefer to keep them the same unless there's a known reason. @jeanregisser have there ever been any issues with us under-estimating the gas needed for send?

Reply:
No issue that I know, we already use a global infation factor.

https://github.com/valora-inc/wallet/blob/268b65d121c2038b46dc3778ff35bcd51bb3259d/src/config.ts#L32